_posts/2016-03-16-AngularTS.md

```diff
@@ -0,0 +1,413 @@
+---
+layout: post
+author: kevin_van_houtte
+title: 'AngularTS: A new look @ Angular'
+image: /img/angularts.jpg
+tags: [TypeScript, AngularJS]
+category: AngularJS, TypeScript
+comments: true
+---
+
+## Combining the best of two worlds.
+
+Since my introduction to the heroic AngularJS framework at Devoxx around 4 years ago.
+I was intrigued and set for an adventure.
+With Angular 2 release in sight we have to prepare ourselves with the migrating road map coming up.
+One of the core changes in angular, was changing the code language to TypeScript.
+This blog will cover the use of angular components in TypeScript.
+But what is this TypeScript? 
+TypeScript is a superset of JavaScript that focuses on strong typing and the new ES6 features: classes, interfaces and modules.
+Like in common OO languages like Java and C# these features aren’t new.
+These features gives the developer the opportunity to build a object oriented architecture in JavaScript.
+With that in mind let’s see what the advantages are: 
+
+### Transpiling
+The DOM on your website can only recognize JavaScript.
+With this said they had to come with a way to compile TS.
+Because TS is a superset of JS it can transpile to plain JavaScript before including it into HTML.
```

Comment:
TypeScript is a compile. So change transpile to compile (you mentioned this earlier in the blog post).